### ISSUE TITLE
Maybe a bug in NetworkVisualization-TensorFlow.py of Assignment 3 

### ISSUE THREAD

**Issue by LiUzHiAn** [hungduyle/CS231n-2018Spring#3]
Hi,dhungle!

In the class visualization part, we want to generate a image to maxmize the loss which includes the score value (we got from CNN model) and regularization part. Right?

But in your implementation, you wrote as following: `loss = model.scores[0, target_y] - l2_reg * tf.reduce_sum(model.scores**2)`, I think it should be `loss = model.scores[0, target_y] - l2_reg * tf.reduce_sum(model.image**2)`  cause the regularization part of loss is for generated image.

You might get a nicer picture after modifying this. [my pictures](https://github.com/LiUzHiAn/cs231n-2018-solutions/blob/master/assignment3/NetworkVisualization-TensorFlow.ipynb)

**Comment by dhungle** [comment]
Dear LuizhiAn,

You are right! Thank for pointing that out. I'll update the code accordingly.

For the moment, I'll close this issue.
